README.adoc

```diff
@@ -0,0 +1,124 @@
+= se-edu-bot
+:toc: preamble
+:toc-title:
+:sectnums:
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+endif::[]
+ifdef::env-github,env-browser[:outfilesuffix: .adoc]
+
+Helps out with managing SE-EDU projects.
+
+== Local development setup
+
+=== Prerequisites
+
+. Node `7.9.0` or later.
+. An editor with Typescript language service support is strongly recommended.
+  (e.g. https://code.visualstudio.com/[VS Code])
```

Comment:
Maybe add a `Recommended editors` section.

Note that for VSCode, the `editorconfig` plugin must be installed for it to respect `.editorconfig` files.